install.ps1

```diff
@@ -1,43 +1,43 @@
+#!/usr/bin/env powershell
```

Comment:
where did you see `powershell`? I think it is called `pwsh`?

Did you test this?
If you run `./install.ps1` it will tell you if it works.